bigchaindb/__init__.py

```diff
@@ -73,6 +73,7 @@
         'fmt_console': log_config['formatters']['console']['format'],
         'fmt_logfile': log_config['formatters']['file']['format'],
         'granular_levels': {},
+        'benchmark': False
```

Comment:
Instead of having a flag to enable benchmark it would be better to enable it via log level instead i.e. something as following

```
    "level_logfile": "benchmark",
```